src/avkit.cs

```diff
@@ -544,9 +688,10 @@ public enum AVRoutePickerViewButtonStyle : long {
 		Custom,
 	}
 
+#endif
 	interface IAVRoutePickerViewDelegate { }
 
-	[TV (11,0), iOS (11,0)]
+	[TV (11,0), iOS (11,0), Mac (10,15)]
 	[Protocol, Model]
```

Comment:
I think I should add ```Model (AutoGeneratedName = true)```here to use ```IAVRoutePickerViewDelegate```right ?

Reply:
No, don't add `AutoGeneratedName = true` to existing protocols, it's a breaking change.